model/supervised_finetuning/trainer.py

```diff
@@ -92,6 +94,8 @@ def argument_parsing(notebook=False, notebook_args=None):
     parser.add_argument("--local_rank", type=int, default=-1)
     parser.add_argument("--deepspeed", action="store_true")
     parser.add_argument("--no-deepspeed", dest="deepspeed", action="store_false")
+    parser.add_argument("--poly_eps", type=float, default=1.0)
+    parser.add_argument("--seq2seq_model", action="store_true")
```

Comment:
I would add these to the config files

Reply:
> I would add these to the config files

Yes, normally it was but because of the file problems I had, it seems like that part was not updated.